src/main/java/edu/umich/lib/dor/replicaexperiment/service/ReplicaService.java

```diff
@@ -29,4 +31,23 @@ public Replica createReplica(InfoPackage infoPackage, Repository repository) {
     public List<Replica> getReplicas() {
         return replicaRepo.findAll();
     }
+
+    public Replica getReplica(
+        InfoPackage infoPackage, Repository repository
+    ) {
+        var matchingReplicas = replicaRepo.findAllByInfoPackage(infoPackage)
+            .stream()
+            .filter(r -> r.getRepository().getId() == repository.getId())
+            .toList();
+        return matchingReplicas.getFirst();
```

Comment:
There is an assumption so far in this application that pairs of an `InfoPackage` and a `Repository` are unique, i.e. you can't have more than one replica of an `InfoPackage` in a single repository. This might operationally be necessary if we want to use package identifiers as `id`s for OCFL.